api/src/main/java/javax/config/spi/ConfigSource.java

```diff
@@ -131,4 +134,14 @@ default int getOrdinal() {
      */
     String getName();
 
+    /**
+     * This callback should get invoked if an attribute change got detected.
+     *
+     * @param reportAttributeChange will be set by the {@link ConfigSourceProvider} after this
```

Comment:
A ConfigSource does not necessarily belong to a ConfigSourceProvider.
If a ConfigSource is loaded using a ServiceLoader, who is responsible to call this method? The ConfigBuilder?

Reply:
No, the created Config instance.